charts/node/README.md

```diff
@@ -230,14 +230,16 @@ If you're running a collator node:
 | `node.perNodeServices.relayP2pService.annotations`                        | Annotations to be set on relay-chain P2P Services                                                                                                                                                                                                    | `{}`                                                                |
 | `node.perNodeServices.relayP2pService.port`                               | Port to use to expose the relay-chain P2P Service externally                                                                                                                                                                                         | `9944`                                                              |
 | `node.perNodeServices.relayP2pService.ws.enabled`                            | If true, creates a relay-chain P2P Service for every node in the statefulset exposing the WebSocket P2P endpoint                                                                                                                          | `false`                                                              |
-| `node.perNodeServices.relayP2pService.ws.port`                            | Port to use to expose the relay-chain WebSocket P2P Service externally                                                                                                                          | `30334`                                                              |
```

Comment:
Please add the following breaking changes in the README when moving to this version